pkg/services/generic.go

```diff
@@ -154,6 +155,16 @@ func (s *sqlGenericService) buildSearch(listCtx *listContext, d *dao.GenericDao)
 		return true, nil
 	}
 
+	if isJSONBSearch(listCtx.args.Search) {
+		// focus on jsonb, ignore type = 'Single' or 'Bundle'
+		jsonbSearch, _, _ := strings.Cut(listCtx.args.Search, "and type=")
```

Comment:
what does "and type=" mean?

Reply:
it was added at the beginning to decide the type is `Single` or `Bundle`. refer to https://github.com/openshift-online/maestro/blob/7a3daed319b16acc20ee3a34f6a8e97c533e253e/pkg/handlers/resource.go#L104. Here just focus on jsonb search